ynr/settings/base.py

```diff
@@ -332,12 +286,13 @@ def root(*path):
     Please don't quote third-party candidate sites \u2014 we prefer URLs of
     news stories or official candidate pages."""
 
-# By default, cache successful results from MapIt for a day
+# By default, cache successful results from Every Election for a day
 EE_CACHE_SECONDS = 86400
+
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.postgresql",
-        "NAME": os.environ.get("DATABASE_NAME", "ynr"),
+        "NAME": os.environ.get("POSTGRES_DBNAME", ""),
```

Comment:
I've gone ahead and set `POSTGRES_DBNAME` in Parameter Store for dev/staging/production
Trying to start django with `DATABASES["default"]["NAME"] = ""` will throw `ImproperlyConfigured` so `POSTGRES_DBNAME` is effectively a required setting.